src/tests/ftest/deployment/basic_checkout.py

```diff
@@ -20,16 +19,45 @@
     :avocado: recursive
     """
 
-    def test_basic_checkout(self):
+    def test_basiccheckout_sanity(self):
         """
         Test Description: Bundles four tests into one and run in the
                           following sequence - ior_small, mdtest_small,
                           ec_smoke and autotest.
         :avocado: tags=all,deployment,full_regression
         :avocado: tags=hw,large
         :avocado: tags=dfuse,ior,mdtest
-        :avocado: tags=basiccheckout
+        :avocado: tags=basiccheckout,basiccheckout_sanity
         """
+
+        # ior easy
+        self.run_performance_ior(namespace="/run/ior_dfs_sx/*", use_stonewalling_read=True)
+        self.run_performance_ior(namespace="/run/ior_dfs_ec_16p2gx/*", use_stonewalling_read=True)
```

Comment:
(pylint-unexpected-keyword-arg) Unexpected keyword argument 'use_stonewalling_read' in method call